### PR TITLE
add export function; reorganize imports

### PR DESCRIPTION
**Pull request by kreuzert**
export function:
copy files from `_job_/csv/2020_09_11` to `shared_folder/csv/2020_10_06`

This way we will get rid of the offset, without changing anything within the computation

### CODE REVIEW

**Review by pnieters — CHANGES_REQUESTED** [review]
I think you were 1 commit behind -- I added an indicator for 'RKI data above 95th percentile' for unusually high data; useful for plotting. Requires rki data before dataframe construction (108-110), in addition to the extra df column.
Please remerge current file into your version!

File: src/results_to_csv.py [hunk 1]
```diff
@@ -103,11 +110,8 @@ def plotdata_csv(start, n_weeks, csv_path, counties, output_dir):
         data=prediction_quantiles_trend[95],
         index=ext_index,
         columns=target.columns)
-    
+
     for (county, county_id) in countyByName.items():
-        
-        rki_data = np.append(target.loc[:, county_id].values, np.repeat(np.nan, 5))
```
Comment (pnieters) [inline src/results_to_csv.py:109]: Need this calculated before df

File: src/results_to_csv.py [hunk 2]
```diff
@@ -119,17 +123,33 @@ def plotdata_csv(start, n_weeks, csv_path, counties, output_dir):
             'Trend Prediction Q75': prediction_q75_trend.loc[:,county_id].values,
             'Trend Prediction Q5': prediction_q5_trend.loc[:,county_id].values,
             'Trend Prediction Q95': prediction_q95_trend.loc[:,county_id].values,
-            'RKI Meldedaten': rki_data,
+            'RKI Meldedaten': np.append(target.loc[:,county_id].values, np.repeat(np.nan, 5)),
             'is_nowcast': (day_m5 <= ext_index) & (ext_index < day_0),
-            'is_prediction': (day_0 <= ext_index),
-            'is_high': np.less(prediction_q95_trend.loc[:,county_id].values, rki_data)},
```
Comment (pnieters) [inline src/results_to_csv.py:125]: removed on purpose?

**Comment by kreuzert** [comment]
Used the code on JUSUF at $PROJECT_.../autostart_csv/BSTIM-Covid19 as base. I assume that code was not up to date

**Comment by pnieters** [comment]
Cheers!

**Pull request merged**